infra/repo_manager.py

```diff
@@ -109,7 +109,21 @@ def get_current_commit(self):
       The current active commit SHA.
     """
     out, _, _ = self.git(['rev-parse', 'HEAD'], check_result=True)
-    return out.strip('\n')
+    return out.strip()
+
+  def get_parent(self, commit, count):
```

Comment:
maybe just name this variable "n" instead of count and the comment will be easier to understand.